ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -36,6 +43,10 @@ export class Send extends IronfishCommand {
       char: 'o',
       description: 'The fee amount in IRON',
     }),
+    feeRate: Flags.string({
+      char: 'r',
+      description: 'The fee rate amount in IRON/Kilobyte',
```

Comment:
nit: `IRON/KB` is what you wrote in other places in our code.

Reply:
this is the external message to user, I want to be clear since some user might not aware KB is kilobyte in this context.